_posts/2023-07-05-quarkus-3-2-0-final-released.adoc

```diff
@@ -33,6 +34,14 @@ Once you upgraded to 3.0, also have a look at the https://github.com/quarkusio/q
 
 == What's new?
 
+=== Build time analytics
+
+To be able to better understand how Quarkus is used, we have added a new build time analytics feature. It is anonymous and opt-in thus fully voluntary.
+
+On first startup of `quarkus dev` you will be asked if you want to share build time analytics with Quarkus community. If you answer yes, then on `quarkus dev` and builds a small package is sent with info about java, maven/gradle, quarkus versions, public extensions used, build time, etc. This will help us to better understand how Quarkus is used and what we can improve. If you answer no, then no data is sent.
```

Comment:
```suggestion
On first startup of `quarkus dev` you will be asked if you want to share build time analytics with the Quarkus community. If you answer yes, then on `quarkus dev` and builds a small package is sent with information about Java, Maven/Gradle, Quarkus versions, public extensions used, build time, etc. This will help us to better understand how Quarkus is used and what we can improve. If you answer no, then no data is sent.
```